api/external/envoy/api/v2/discovery.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package envoy.api.v2;
+package solo.io.envoy.api.v2;
```

Comment:
Does this make it a breaking change? Now the extauth and ratelimit xds clients will need to update their inputs/outputs

Reply:
Ah I hadn't thought about that. I did end up having to make changes on the Gloo side to get this working for them. Do you suspect this will be a breaking change from a client's perspective as well? I'm happy to undo it, just thought it made sense as a change since all of our other copies protos have that package

Reply:
No I think it's good to update, i just wanted to point out that this should now be a BREAKING_CHANGE PR and a minor version bump